### PR TITLE
BLUEBUTTON-1157: Correcting prod and prod-sbx to load the correct client certs.

### PR DESCRIPTION
**Pull request by jzulim**
# Why
We need the correct client certs installed for each environment. I goofed and a copy/paste issue is causing us to deploy certs configured for the TEST environment to all our environments. 

# What

Updated variable data_server_ssl_client_certificates for both prod-sbx and prod to reflect the correct client certs.

### CODE REVIEW

File: ops/ansible/playbooks-ccs/vars/prod-sbx/group_vars/all/env_specific.yml [hunk 1]
```diff
@@ -1,42 +1,51 @@
...
 
 # Bucket where deployment artifacts are located. 
 s3_bucket_deployment: 'bfd-mgmt-artifacts-577373831711'
 
 # Name of ETL Staging Bucket 
-s3_bucket_etl_staging: 'bfd-prod-sbx-etl-577373831711'
-
-# The abbreviated name for this environment, per the naming conventions used by HealthAPT.
-# env_name: 'ts'
+s3_bucket_etl_staging: 'bfd-prod-etl-577373831711'
```
Comment (RickHawesUSDS) [inline ops/ansible/playbooks-ccs/vars/prod-sbx/group_vars/all/env_specific.yml]: Why this change for the ETL staging bucket? Why use a `prod` named bucket in the prod-sbx vars?

Comment (jzulim) [reply ops/ansible/playbooks-ccs/vars/prod-sbx/group_vars/all/env_specific.yml]: fixed. thx.
